RvmSharp/Utils/StreamExtensions.cs

```diff
@@ -0,0 +1,17 @@
+namespace RvmSharp.Utils;
+
+using System;
+using System.IO;
+
+internal static class StreamExtensions
+{
+#if DOTNET7_0_OR_GREATER
+    // 'ReadExactly' is implemented in Stream in .NET 7.0 and later
+#else
```

Comment:
Do we need to support older that dotnet 7?

Reply:
Keeping this for now to have Unity support in the rvmsharp package, but this may be removed at some point as we dont have any current users of it.